app/theme.ts

```diff
@@ -13,82 +14,16 @@ const customTheme = extendTheme({
       secondary: "#db0a40",
       accentGreen: "#32CD32",
       accentGrey: "#787373",
+      accentOrange: "#E9981F",
+      accentYellow: "#FAD50AD4",
     },
   },
   fonts: {
     heading: "var(--font-roboto-slab)",
     body: "var(--font-lato)",
   },
   components: {
-    Link: {
-      variants: {
-        buttonPrimaryDark: {
-          color: "white",
-          backgroundColor: "brand.secondary",
-          borderRadius: "20px",
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-          padding: "6px",
-          fontWeight: "700",
-          _hover: {
-            color: "brand.text",
-            transitionDuration: "300ms",
-            textDecoration: "none",
-          },
-        },
-        buttonPrimaryLight: {
-          color: "white",
-          backgroundColor: "brand.secondary",
-          borderRadius: "20px",
-          border: "solid 2px",
-          borderColor: "white",
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-          padding: "6px",
-          fontWeight: "700",
-          _hover: {
-            borderColor: "black",
-            transitionDuration: "300ms",
-            textDecoration: "none",
-          },
-        },
-        buttonSecondary: {
-          color: "brand.text",
-          backgroundColor: "white",
-          borderRadius: "20px",
-          display: "flex",
-          alignItems: "center",
-          justifyContent: "center",
-          padding: "6px",
-          fontWeight: "700",
-          _hover: {
-            backgroundColor: "brand.secondary",
-            transitionDuration: "300ms",
-            textDecoration: "none",
-          },
-        },
-        navigation: {
-          color: "white",
-          fontSize: "24px",
-          _hover: {
-            borderBottom: "solid 2px",
-            borderBottomColor: "brand.secondary",
-            textDecoration: "none",
-            transitionDuration: "300ms",
-          },
-        },
-        footer: {
-          _hover: {
-            borderBottom: "solid 2px",
-            borderBottomColor: "brand.secondary",
-            textDecoration: "none",
-            transitionDuration: "300ms",
-          },
-        },
-      },
-    },
+    Link: Link,
```

Comment:
Moved Link theme to a separate file and added some missing styles for different link variants